### ISSUE TITLE
Updating vuepress-plugin-live breaks docs examples

### ISSUE THREAD

**Issue by TheJaredWilcurt** [AustinGil/vuetensils#78]
Updating `vuepress-plugin-live` from `1.4.2` to `1.5.0+` breaks the following examples in the docs:

* http://localhost:8080/components/Async.html#call-promise-using-refs
* http://localhost:8080/components/Dialog.html#prevent-scrolling
* http://localhost:8080/components/Drawer.html#styled-examples
* http://localhost:8080/components/Drawer.html#examples
* http://localhost:8080/components/Drawer.html#using-v-model
* http://localhost:8080/components/Drawer.html#right-aligned
* http://localhost:8080/components/Drawer.html#prevent-page-scroll
* http://localhost:8080/components/Drawer.html#add-transitions
* http://localhost:8080/components/File.html#custom-dropzone-content
* http://localhost:8080/components/Form.html#styled-example
* http://localhost:8080/components/Form.html#form-status
* http://localhost:8080/components/Form.html#clearing-inputs
* http://localhost:8080/components/Input.html#styled-example
* http://localhost:8080/components/Intersect.html#using-a-scoped-slot
* http://localhost:8080/components/Resize.html#styled-example
* http://localhost:8080/components/Toggle.html#scoped-slots
* http://localhost:8080/directives/clickout.html#usage
* http://localhost:8080/directives/intersect.html#default-behavior

**[VuePress-Plugin-Live Version Release Notes](https://github.com/vue-styleguidist/vuepress-plugin-live/releases)**

**Comment by Stegosource** [comment]
Interesting. Im going to have to spend some time with this one.

**Comment by TheJaredWilcurt** [comment]
Specifically happens when going from v1.4.2 to the next release, 1.5.0 (or above).


* [v1.4.2 to v1.5.0](https://github.com/vue-styleguidist/vuepress-plugin-live/compare/v1.4.2...v1.5.0)

Not much changed between these two, so hopefully it is easy to figure out.

One change that occurred was updating a dependency, Vue-Live, from [v1.4.3 to v1.6.0](https://github.com/vue-styleguidist/vue-live/compare/v1.4.3...v1.6.0). So a code change in there may be related also.

**Comment by lbineau** [comment]
Hello,
Is there any update on that one? I saw the `plugin:[['live']]` is missing on vuepress config I wonder if a newer version could have fixed that issue so we can reenable it or if there are alternatives to vuepress live plugin.
Not having a live example is quite painful when looking at the documentation.

**Comment by AustinGil** [comment]
The original issue was actually unrelated to this latest one, but I've upgraded Vuepress to the next version. Unfortunately, vuepress-plugin-live does not currently support the next version. I've reached out to the authors to see if they are going to support it and waiting to hear back. If not, then my options are to either make a PR myself (or fork it) or maybe use an embeded Codesandbox. It may take me a little while to get to it, but I fully intend to have live examples in the documentation.
